src/sql/statement.rs

```diff
@@ -669,13 +670,21 @@ fn handle_create_sink(scx: &StatementContext, stmt: Statement) -> Result<Plan, f
     // Validate that we can actually encode this stream as Avro.
     let relation_desc = catalog_entry.desc()?.clone();
     let _ = interchange::avro::encode_schema(&relation_desc)?;
+    let topic_name = format!(
+        "{}-{}",
+        topic,
+        SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .expect("Failed to get system time while creating sink")
+            .as_secs()
+    );
```

Comment:
I know this is WIP, but as a reminder for us for later: planning should be deterministic, so this should depend on some per-boot state that's stored in the catalog.

Reply:
yup -- that + the admin client bit are what I'm banging out rn